### PR TITLE
Paginate registry results

### PR DESCRIPTION
**Pull request by appilon**
The verified partner providers was getting close to the max page size (100)

### CODE REVIEW

**Comment by appilon** [comment]
I don't see how the changes I made would affect the tests, I suspect a race in some of our unit tests, specifically around shutdown, I can't recreate locally on darwin

**Review by aeschright — APPROVED** [review]
👍

File: internal/schemas/gen/gen.go [hunk 1]
```diff
@@ -50,12 +50,14 @@ func gen() error {
 	if err != nil {
 		return err
 	}
+	log.Printf("len=%d", len(providers))
```
Comment (radeksimko) [inline internal/schemas/gen/gen.go:53]:
```suggestion
	log.Printf("fetched official providers: %d", len(providers))
```

File: internal/schemas/gen/gen.go [hunk 2]
```diff
@@ -50,12 +50,14 @@ func gen() error {
...
 
 	log.Println("fetching verified partner providers from registry")
 	partnerProviders, err := listProviders("partner")
 	if err != nil {
 		return err
 	}
+	log.Printf("len=%d", len(partnerProviders))
```
Comment (radeksimko) [inline internal/schemas/gen/gen.go:60]:
```suggestion
	log.Printf("fetched partner providers: %d", len(partnerProviders))
```

**Review by radeksimko — APPROVED** [review]

**Pull request merged**